main.js

```diff
@@ -79,34 +79,44 @@ projects.forEach((project) => {
         <div class="card">
             <img src="images/${project.featureImage}" alt="Project ${project.id}">
             <div class="card-body">
-                <h5 class="card-title text-center">${project.name}<br> Gain+Glory</h5>
+                <h5 class="card-title text-center">${project.name}<br> ${project.company}</h5>
                 <div class="d-flex justify-content-around">
                     <button class="btn btn-secondary btn-outline-success px-1 rounded-3" disabled>${project.tools[0]}</button>
                     <button class="btn btn-secondary btn-outline-success px-1 rounded-3" disabled>${project.tools[1]}</button>
                     <button class="btn btn-secondary btn-outline-success px-1 rounded-3" disabled>${project.tools[2]}</button>
                     <button class="btn btn-secondary btn-outline-success px-1 rounded-3" disabled>${project.tools[3]}</button>
                 </div>
                 <div class="text-center">
-                    <button type="button" class="btn btn-success m-3 showModal" data-bs-toggle="modal" data-bs-target="#project1">See Project</button>
-                    <div class="modal fade" id="project1">
-                        <div class="modal-dialog modal-fullscreen">
+                    <button type="button" class="btn btn-success m-3 showModal" data-bs-toggle="modal" data-bs-target="#project${project.id}">See Project</button>
+                    <div class="modal fade" id="project${project.id}">
+                        <div class="modal-dialog bg-white rounded-3">
```

Comment:
- [ ] Good job trying to add the white background :+1: : , I suggest trying to stick to the design a bit more, just as requested by the previous reviewer. Please make sure you have the same proportion in terms of width and height as per the design
- [ ] Kindly make sure you use the same font and color as the design.
Your design (you can refer to this :width: 876px;
height: 151px;

font-family: 'Inter';
font-style: normal;
font-weight: 400;
font-size: 16px;
line-height: 30px;
/* or 188% */


/* Neutrals/N600 */

color: #344563;
)

![yourWidth](https://user-images.githubusercontent.com/97834160/209157263-742f4ed5-846a-4455-9228-12062201b801.png)

Figma:
![image](https://user-images.githubusercontent.com/97834160/209157075-69c71efd-b690-4e97-970d-5a8996b6adbb.png)

Reply:
> * [x]  Good job trying to add the white background 👍 : , I suggest trying to stick to the design a bit more, just as requested by the previous reviewer. Please make sure you have the same proportion in terms of width and height as per the design
> 
>     * [x]  Kindly make sure you use the same font and color as the design.
>       Your design (you can refer to this :width: 876px;
>       height: 151px;
> 
> 
> font-family: 'Inter'; font-style: normal; font-weight: 400; font-size: 16px; line-height: 30px; /* or 188% */
> 
> /* Neutrals/N600 */
> 
> color: #344563; )
> 
> ![yourWidth](https://user-images.githubusercontent.com/97834160/209157263-742f4ed5-846a-4455-9228-12062201b801.png)
> 
> Figma: ![image](https://user-images.githubusercontent.com/97834160/209157075-69c71efd-b690-4e97-970d-5a8996b6adbb.png)

Please clear whether these suggestions about modal or body ?

Reply:
@GedeonTS  please clear.

Reply:
Hi @DilsherB, all the suggestions that are mandatory are about the modal for now, I see you made some changes :eyes: :+1: But it is good to check back your whole project to see if it sticks to the Figma, for learning purposes.